app/assets/json/translations/react/en.json

```diff
@@ -47,9 +47,14 @@
   "listings.atTotalIncome": "at <span class='eligibility-subtext'>$%{income}</span> %{per}",
   "listings.availableUnits": "Available Units",
   "listings.availableUnitsAndOpenWaitlist": "Available Units & Open Waitlist",
+  "listings.availableUnitsAndWaitlist": "Available units and waitlist",
+  "listings.availableUnitsAndWaitlistDescription": "Once lottery winners fill all available units, additional lottery winners will be placed on the waitlist for %{number} %{unit}",
+  "listings.availableUnitsAndWaitlistDescription.pluralUnits": "units",
+  "listings.availableUnitsAndWaitlistDescription.singularUnit": "unit",
```

Comment:
Concatenating text props is usually not a good idea. This could be simplified to 2 text props instead of 3, but the 0 units scenario is also awkward because with the current logic it looks like the text would be `...additional lottery winners will be placed on the waitlist for 0 unit`. Is this a possible scenario?

I am inclined to just remove the number of units from this text
![image](https://user-images.githubusercontent.com/98352600/167923897-4710af55-2c63-4272-a490-49ddc554e0da.png)

Reply:
This change was to solve an existing `1 units` bug - open to removing the units text if Yindi/Emily are!

Reply:
@yindipei @emilyshore  is it ok to remove the number of units within the paragraph? see above screenshot

Reply:
Assuming this is the only instance of this situation, we will not be compromising the UX if we remove the italicized part. My answer is Yes! It is okay to remove the number of units within the paragraph.